packages/console/src/pages/EnterpriseSsoDetails/Settings/LogosUploader/index.tsx

```diff
@@ -10,13 +11,15 @@ import type { FormType } from '../index.js';
 
 import * as styles from './index.module.scss';
 
+const allowedMimeTypes: AllowedUploadMimeType[] = ['image/png', 'image/jpeg', 'image/svg+xml']; // Only allow `svg`, `png`, `jpg` and `jpeg` files.
+
 function LogosUploader() {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const [uploadLogoError, setUploadLogoError] = useState<string>();
   const [uploadDarkLogoError, setUploadDarkLogoError] = useState<string>();
 
   const { control } = useFormContext<FormType>();
-  const { description } = useImageMimeTypes();
+  const { description } = useImageMimeTypes(allowedMimeTypes);
```

Comment:
Nit: give `useImageMimeTypes` a better name.

Reply:
This aligns with the use of this hook in other places, can keep it.